test/gen_batch_server_SUITE.erl

```diff
@@ -145,6 +147,30 @@ cast_many(Config) ->
     ?assert(meck:validate(Mod)),
     ok.
 
+cast_batch(Config) ->
+    Mod = ?config(mod, Config),
+    meck:new(Mod, [non_strict]),
+    meck:expect(Mod, init, fun(Init) -> {ok, Init} end),
+    Args = #{},
+    {ok, Pid} = gen_batch_server:start_link({local, Mod}, Mod, Args, []),
+    Self = self(),
+    meck:expect(Mod, handle_batch,
+                fun(Ops, State) ->
+                        {cast, {put, K, V}} = lists:last(Ops),
+                        ct:pal("cast_batch: batch size ~b~n", [length(Ops)]),
```

Comment:
Was this left in intentionally?

Reply:
@michaelklishin this is similar to the `cast_many` test, but can be removed if needed :-)

Reply:
I don't know if it was kept there before committing intentionally. We occasionally have such messages in CT suites but only for truly complex tests and with the low priority option. Can we remove them here?

Reply:
i can remove both messages if @kjnilsson is OK . Just let me know :)

Reply:
@michaelklishin i've removed both messages in separate commit. if  @kjnilsson think the message needs to be kept i can revert 3f35b955b824c51a2b9df134dbb2abf2df1877b8 .